pkg/verity/verity.go

```diff
@@ -139,9 +137,9 @@ func isCryptsetupEINVAL(err error) bool {
 	return ok && cse.Code() == -22
 }
 
-var cryptsetupTooOld = errors.Errorf("libcryptsetup not new enough, need >= 2.3.0")
+var CryptsetupTooOld = errors.Errorf("libcryptsetup not new enough, need >= 2.3.0")
```

Comment:
Do we already check this via `stacker check` ? if not we should there; would be a bummer to find out that we can't generate verity data after already creating an OCI image.

Reply:
Hmmm..we don't. But this check happens inline when creating the fs img.

Reply:
That's unfortunately late ... if your FS image takes a while to build only to find out that you can't create an erofs image due to host issues will be frustrating to users.

Reply:
Let's add a routine in another PR. This one is already super-loaded.

Reply:
https://github.com/project-machine/atomfs/issues/26
^ filed